en/restaurant/restaurant-information.md

```diff
@@ -0,0 +1,77 @@
+---
+lang: en
+title: Restaurant information
+ref: rest-info
+parent: Restaurant's guide
+nav_order: 5
+---
+
+# Restaurant information
+
+The general information of the restaurant is accessible and modifiable at any time on the home page of by clicking on `Edit` button near of the name of the restaurant at the top left of the page.
+
+On this page you can : 
+
+- Change the **image** of your restaurant, which will appear to clients on CoopCycle. In order to do that, you can directly drop an image where it is indicated.
+- Disable or enable your restaurant to appear on CoopCycle. When disabled the restaurant page is not accessible by visitors.
+
+Four tabs are accessible on this page in order to look or modify your informations - please don't forget to click on `save` button at the bottom of the page when you are modifying fields : 
+
+- [General](#General)
+- [Options](#Options)
+- [Fulfillment](#Fulfillment)
+- [Settlement](#Settlement)
+
+## General
+
+On this `General` tab, you can modify general informations about your restaurant :
+
+- Displayed to clients :
+  - Name of the Restaurant on CoopCycle
+  - Type of Business
+  - Adress
+  - Description of your restaurant 
+  - Webd
+  
+- Not displayed to clients :
+  - Website
+  - Telephone number
+  - SIRET number
```

Comment:
SIRET is only for France, replace all subsequent bullet points by "business information (depending on country)"